tutorials/hana-cloud-dl-clients-overview/hana-cloud-dl-clients-overview.md

```diff
@@ -32,70 +33,73 @@ SAP HANA Cloud is composed of multiple components.
 
     ![independent data lake](standalone.png)
 
-    An independent data lake can be configured with additional options such a collation value of UTF8BIN and blank padding set to ON to be more compatible with an on-premise SAP IQ.
-
+    ![Standalone data lake](standalone2.png)
 
-1. Using the SAP BTP Cockpit within the [SAP BTP Trial](https://account.hanatrial.ondemand.com/trial/#/home/trial), navigate to subaccount, the space, and then to the **SAP HANA Cloud** section as shown below.
+    A standalone data lake can be configured with additional options such a collation value of UTF8BIN and blank padding set to ON to be more compatible with an on-premise SAP IQ.
 
-    ![SAP BTP Cockpit](BTP-cockpit.png)
+1. Using the SAP BTP Cockpit open SAP HANA Cloud Central using one of the two methods described in the step Choose where to deploy the database instances.
 
-2. Click on **Manage SAP HANA Cloud** to open SAP HANA Cloud Central, which is used to create and configure SAP HANA Cloud instances.
-
-    ![SAP BTP Cockpit](BTP-cockpit2.png)
-
-3. If a data lake is not already present, add one using one of the three methods previously described.
+2. If a data lake is not already present, add one using one of the three methods previously described.
 
     Take note when creating the data lake that the administration user is HDLADMIN.
 
     >The HDLADMIN user has a [login policy](https://help.sap.com/viewer/745778e524f74bb4af87460cca5e62c4/latest/en-US/a43f448484f21015924f9951e9b77e32.html) that enforces the [update of the password](https://help.sap.com/viewer/745778e524f74bb4af87460cca5e62c4/latest/en-US/a458edd784f2101580a1eca5042678f8.html) after 180 days.  
 
-    >---
-
-    >It is also possible to create and manage data lake instances using the [Cloud Foundry CLI](https://help.sap.com/viewer/9ae9104a46f74a6583ce5182e7fb20cb/hanacloud/en-US/921f3e46247947779d69b8c85c9b9985.html).
-
-4. As this is a trial account, set allowed connections to **Allow all IP addresses** so that client applications can connect from any IP address.  
+3. As this instance is a free tier or trial account, set allowed connections to **Allow all IP addresses** so that client applications can connect from any IP address.  
 
     ![Allowed connections](allowed-connections.png)
 
-5. After a while, press the **Refresh** button or enable auto-refresh and the status will change from CREATING to RUNNING.
+4. After a while, press the **Refresh** button or enable auto-refresh and the status should change from CREATING to RUNNING once the instance has been created.
 
     ![data lake running](data-lake-running.png)
 
-    >Note that the SAP HANA Cloud, HANA data lake trial or free tier instances are shut down overnight and will need to be restarted before working with them the next day.
+    >Note that the SAP HANA Cloud, HANA data lake free tier or trial instances are shut down overnight and will need to be restarted before working with them the next day.

```

Comment:
Please rephrase and do not start a note with the keyword "Note", but use a label instead, for example, \*\*Important:\*\*